src/main/java/eu/eutampieri/catacombs/model/Face.java

```diff
@@ -0,0 +1,20 @@
+package eu.eutampieri.catacombs.model;
+
+public enum Face {
```

Comment:
I would remove reduntant FACE_ from every name and maybe call the whole enum Directon instead of Face.